calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -159,10 +163,13 @@ def to_json(self):
             json_dict["ipip"] = "tunl0"
         if self.masquerade:
             json_dict["masquerade"] = True
-        # Only write IPAM when set to False, this keeps the behavior the same
-        # for old versions of the data.
+        # Only write "ipam" and "disabled" when they differ from their default
+        # values.  This keeps the interface unchanged between versions when
+        # these fields are not required.
         if not self.ipam:
             json_dict["ipam"] = False
+        if self.disabled:
```

Comment:
Is it really a problem if we have a new key in the dictionary? Old clients will just ignore it.